doc/classes/DisplayServer.xml

```diff
@@ -1612,7 +1612,7 @@
 			<param index="0" name="min_size" type="Vector2i" />
 			<param index="1" name="window_id" type="int" default="0" />
 			<description>
-				Sets the minimum size for the given window to [param min_size] (in pixels). Normally, the user will not be able to drag the window to make it larger than the specified size. See also [method window_get_min_size].
+				Sets the minimum size for the given window to [param min_size] (in pixels). Normally, the user will not be able to drag the window to make it smaller than the specified size. See also [method window_get_min_size].
```

Comment:
```suggestion
				Sets the minimum size for the given window to [param min_size] in pixels. Normally, the user will not be able to drag the window to make it smaller than the specified size. See also [method window_get_min_size].
```
Let's normalise this while we're at it, or add parentheses to the other one but I'd say this is preferable